pkg/api/login_handler.go

```diff
@@ -21,12 +21,30 @@ const (
 	JWTCookieName          = "access_token"
 )
 
-func GenerateJWT(secret []byte, userID int, issuedAt, expiresAt time.Time) (string, error) {
+func generateJWT(claims *jwt.StandardClaims, secret []byte) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	return token.SignedString(secret)
+}
+
+// GenerateJWTLogin supports backward compatibilty for creating a login jwt. The audience is set to LoginAudience which is
```

Comment:
This is still not the Go way to write func docs.  The comment should start by saying what it does.  After that finer details may appear.

Reply:
Fixed (I think)